iib/web/api_v1.py

```diff
@@ -396,6 +396,40 @@ def get_builds() -> flask.Response:
         query_params['user'] = user
         query = query.join(Request.user).filter(User.username == user)
 
+    if from_index:
+        request_create_empty_index_alias = aliased(RequestCreateEmptyIndex, flat=True)
+        request_add_alias = aliased(RequestAdd, flat=True)
+        request_rm_alias = aliased(RequestRm, flat=True)
+        request_fbc_operations_alias = aliased(RequestFbcOperations, flat=True)
+
+        query_params['from_index'] = from_index
+        # Get the image id of the image to be searched
+        from_index_result = Image.query.filter_by(pull_specification=from_index).first()
+        if not from_index_result:
+            # if index_image is not found in image table, then raise an error
+            raise ValidationError(f'{from_index} is not a valid index image')
```

Comment:
Minor: Although "not a valid index image" is technically correct, how about using "not a valid from_index"?

Reply:
Changed